CustomersWithProductsAsync/modals/Customer.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace EFCustomerwithProducts.modals
+{
+    public class Customer
+    {
+        [Key] public int CustomerId { get; set; }
```

Comment:
Annotations is okay, its best practice to use "FluentApi" -> overriding OnModelCreating() method. Keep this for now. 
From next time I recommand to use FluentApi